client/my-sites/checkout/checkout/index.jsx

```diff
@@ -399,6 +399,11 @@ export class Checkout extends React.Component {
 			if ( isJetpackNotAtomic && ! isJetpackProduct ) {
 				return `/plans/my-plan/${ selectedSiteSlug }?thank-you&install=all`;
 			}
+			// If we just purchased a Jetpack product, redirect to the my plans page.
+			if ( isJetpackNotAtomic ) {
+				return `/plans/my-plan/${ selectedSiteSlug }`;
```

Comment:
Maybe we could eventually use `?thank-you&install=SOMETHING` here if we want a dedicated setup flow, but I think that's good as-is for now.

Reply:
Yeah I was thinking about this too. 
I asked design to take another look. 

I first started with `?thank-you` but to me it didn't make sense what the user sees in that case. So I got rid of it for now.

Reply:
Totally, just wanted to make sure we've discussed it as a future option to use.